templates/azure-functions/app/entry.client.tsx

```diff
@@ -0,0 +1,4 @@
+import { hydrate } from "react-dom";
+import { RemixBrowser } from "remix";
```

Comment:
can we update the template to use the full imports?

Reply:
Done. I copied the approach from the other templates.